net/iproute2/BUILD

```diff
@@ -2,9 +2,9 @@ if ! in_depends $MODULE db; then
   sedit 's/arpd//' misc/Makefile
 fi &&
 
-./configure &&
-make &&
-prepare_install &&
-make SBINDIR=/usr/sbin install &&
+OPTS+=" --sbindir=/usr/bin" &&
```

Comment:
This change will unfortunately break all static ip configurations where "lnet" have been used. For now it should remain in /usr/sbin.